### PR TITLE
Fixed a bug in the creation of the result due to missing slashes.

### PR DESCRIPTION
**Pull request by TwoOfTwelve**
For source roots without sub-directories, the files in the results.zip had the wrong name, because there was a slash missing.

### CODE REVIEW

File: core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java [hunk 1]
```diff
@@ -112,7 +112,7 @@ private void copySubmissionFilesToReport(JPlagResult result) {
                 if (relativeFilePath.isEmpty()) {
                     relativeFilePath = file.getName();
```
Comment (Kr0nox) [inline core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java:113]: We could also just preface the files name with a `/`. Would be shorter, simpler solution. 
What do we prefer?

**Review by uuqjz — COMMENTED** [review]
Issues are solved

File: core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java (hunk 1 above)
Comment (tsaglam) [reply core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java:113]: To be honest, I have no real preference. But related to pre-release panic, I say fewer changes are better! :D

**Review by tsaglam — COMMENTED** [review]
Just one question.

File: core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java (hunk 1 above)
Comment (tsaglam) [reply core/src/main/java/de/jplag/reporting/reportobject/ReportObjectFactory.java:113]: No, I would say it is fine as it is right now.

File: core/src/main/java/de/jplag/reporting/FilePathUtil.java [hunk 2]
```diff
@@ -26,4 +26,23 @@ public static String getRelativeSubmissionPath(File file, Submission submission,
...
+        while (rightStripped.startsWith("/")) {
+            rightStripped = rightStripped.substring(1);
+        }
+
+        String leftStripped = left;
+        while (leftStripped.endsWith("/")) {
+            leftStripped = leftStripped.substring(0, leftStripped.length() - 1);
+        }
+
+        return leftStripped + "/" + rightStripped;
```
[the remark this answers is not in the document]
Comment (tsaglam) [reply core/src/main/java/de/jplag/reporting/FilePathUtil.java:46]: Thanks for clarifying. In that case, let us use a constant `ZIP_PATH_SEPARATOR` in that class for it, this allows us to clarify the concept and even add a comment along the lines of `// must always be a forward slash`.

**Review by tsaglam — APPROVED** [review]

**Pull request merged**